backup/setup/migrations.json

```diff
@@ -1,591 +1,568 @@
-		},
-		"state": "up",
-		"name": "remove classes out of teams",
-		"createdAt": {
-			"$date": "2019-12-06T16:30:14.257Z"
-		}
-	},
-	{
-		"_id": {
-			"$oid": "5dc19483d15115187413f3c0"
-		},
-		"state": "up",
-		"name": "remove-broken-class-successor-links",
-		"createdAt": {
-			"$date": "2019-11-05T14:19:26.569Z"
-		},
-		"__v": 0
-	},
-	{
-		"_id": {
-			"$oid": "5d820e442d6f1435214b2d96"
-		},
-		"state": "up",
-		"name": "remove-research-and-dateofresearchconsent",
-		"createdAt": {
-			"$date": "2019-09-17T14:43:38.288Z"
-		},
-		"__v": 0
-	},
-	{
-		"_id": {
-			"$oid": "5ea6d94a5735a85808dbc95a"
-		},
-		"state": "up",
-		"name": "remove-student_edit-from-students",
-		"createdAt": {
-			"$date": "2020-04-27T15:17:39.485Z"
-		},
-		"__v": 0
-	},
-	{
-		"_id": {
-			"$oid": "5d651f8f5e33068c88d5bcce"
-		},
-		"state": "up",
-		"name": "removeRcPws",
-		"createdAt": {
-			"$date": "2019-08-27T12:06:32.917Z"
-		},
-		"__v": 0
-	},
-	{
-		"_id": {
-			"$oid": "5d91b525fee68c5c70d99de1"
-		},
-		"state": "up",
-		"name": "removeUnusedTeamAttributes",
-		"createdAt": {
-			"$date": "2019-09-30T07:46:22.613Z"
-		},
-		"__v": 0
-	},
-	{
-		"_id": {
-			"$oid": "5de6838fc7c4a02707c337dd"
-		},
-		"state": "up",
-		"name": "remove_create_permissions_thuringia",
-		"createdAt": {
-			"$date": "2019-12-02T11:56:47.413Z"
-		},
-		"__v": 0
-	},
-	{
-		"_id": {
-			"$oid": "5d3ef5f4bd2cdb2075fc7e54"
-		},
-		"state": "up",
-		"name": "replace_grandlevel_id_with_number",
-		"createdAt": "2019-07-29T09:57:38.131Z",
-		"__v": 0
-	},
-	{
-		"_id": {
-			"$oid": "5e7f1d63e2c96215c85330f9"
-		},
-		"state": "up",
-		"name": "runIndexSync",
-		"createdAt": {
-			"$date": "2020-03-21T19:35:25.407Z"
-		},
-		"__v": 0
-	},
-	{
-		"_id": {
-			"$oid": "5d26e8c1d0c346ac5bf59bb5"
-		},
-		"state": "up",
-		"name": "schoolyear_19_20_added",
-		"createdAt": {
-			"$date": "2019-07-11T07:44:01.252Z"
-		},
-		"__v": 0
-	},
-	{
-		"_id": {
-			"$oid": "5d442946c6c8392ba30b96a0"
-		},
-		"state": "up",
-		"name": "schoolyear_20_21_added",
-		"createdAt": {
-			"$date": "2019-08-02T12:15:02.584Z"
-		},
-		"__v": 0
-	},
-	{
-		"_id": {
-			"$oid": "5d552b8bc1658a0682fc834a"
-		},
-		"state": "up",
-		"name": "set schoolspecific documentbasedir for instance open",
-		"createdAt": {
-			"$date": "2019-08-15T09:34:59.243Z"
-		},
-		"__v": 0
-	},
-	{
-		"_id": {
-			"$oid": "5d25cf6ddf5c9947bc014236"
-		},
-		"state": "up",
-		"name": "skipRegistrationPermissions",
-		"createdAt": {
-			"$date": "2019-07-10T11:43:41.464Z"
-		},
-		"__v": 0
-	},
-	{
-		"_id": {
-			"$oid": "5edf56430383be55fc9a8ee3"
-		},
-		"state": "up",
-		"name": "updateUserIndexesForLDAP",
-		"createdAt": {
-			"$date": "2020-06-09T09:28:35.874Z"
-		},
-		"__v": 0
-	},
-	{
-		"_id": {
-			"$oid": "5e1c307cfa669ea0863851bc"
-		},
-		"state": "up",
-		"name": "videoconference permissions added in coursesroles",
-		"createdAt": {
-			"$date": "2020-01-13T08:55:24.955Z"
-		},
-		"__v": 0
+[{
```

Comment:
Why is the formatting different here? however, we must add sorting to exported collections as this seems a operating system or version dependant issue, please implement https://github.com/hpi-schul-cloud/schulcloud-server/pull/1344/files#diff-955b40300809ae83a59f34c48353d783R212